raiden_installer/installer_server.py

```diff
@@ -0,0 +1,220 @@
+import sys
+import json
+import time
+import socket
+import webbrowser
+from pathlib import Path
+from web3 import Web3, HTTPProvider
+from eth_utils import to_checksum_address
+from flask import (
+    Flask,
+    render_template,
+    request,
+    redirect,
+    url_for
+)
+from installer_parts import (
+    keystore,
+    raiden_config,
+    funding,
+    raiden
+)
+from constants import (
+    PLATFORM,
+    BINARY_DIR,
+    CONFIG_DIR,
+    KEYSTORE_DIR,
+    TOKEN_AMOUNT,
+    GAS_PRICE,
+    GAS_REQUIRED_FOR_MINT,
+    GAS_REQUIRED_FOR_APPROVE,
+    GAS_REQUIRED_FOR_DEPOSIT
+)
+
+
+def static_assets_absolute_path(static_assets_relative_path: str) -> Path:
+    '''
+    Provides PyInstaller with an
+    absolute path to static files.
+    '''
+    installer_server_file = Path(__file__).resolve()
+    raiden_installer_dir = installer_server_file.parent
+    base_path = getattr(sys, '_MEIPASS', raiden_installer_dir)
+
+    static_assets_absolute_path = Path(base_path).joinpath(
+        static_assets_relative_path
+    )
+    return static_assets_absolute_path
+
+
+if getattr(sys, 'frozen', False):
+    # Installer code running in a bundle
+    template_folder = static_assets_absolute_path('templates')
+    static_folder = static_assets_absolute_path('static')
+
+    app = Flask(
+        __name__,
+        template_folder=template_folder,
+        static_folder=static_folder
+    )
+else:
+    # Installer code running live
+    app = Flask(__name__)
+
+
+@app.route('/', methods=['GET', 'POST'])
+def install_raiden():
+    if request.method == 'POST':
+        keystore_pwd = request.form['keystore-pwd']
+        proj_id = request.form['proj-id']
+        network = 'goerli'
```

Comment:
Do you mean the user input generated from the html form? Or just the variables, `keystore_pwd = None`, like so?

Reply:
I meant the `network = 'goerli'` line. If Görli is the only network we currently support, it would be better to have this indicated in the `constants` module than in the middle of a function that should not be responsible in defining anything.